src/TransactionData.java

```diff
@@ -0,0 +1,83 @@
+public class TransactionData {
+
+    private final Object[] transactionData = new Object[] {
```

Comment:
I think there might be a similar approach we can take, that means we don't have to use the generic `Object[]` type, but rather a `Transaction`.
Let's create two methods:
```java
private Transaction debit(String description, long amount) { /* ... */ }
private Transaction credit(String description, long amount) { /* ... */ }
```
and call these instead of creating `Object` arrays:
```java
debit("Lottery Ticket", 300);
credit("Book Deal", 500000);
//...
```
This lets us create the transactions in an obvious, but still simple manner.